### PR TITLE
next round of discovery edits

### PR DESCRIPTION
**Pull request by duglin**
Basically what we presented during last week's call.

Signed-off-by: Doug Davis <dug@microsoft.com>

### CODE REVIEW

File: discovery/spec.md [hunk 1]
```diff
@@ -58,46 +57,23 @@ This specification defines the following terms:
...
-{ "uri": "https://example.com/discovery/endpoints/123", "name": "MyService" }
-```
+#### Group
 
-This allows for requestors to have some additional information about the
-target resource without the need to perform any additional queries. This
-is most commonly used for user-interfaces that wish to display some meaningful
-information about the resource instead of just its URI. It is RECOMMENDED that
-references include at least the `name` property.
+A set of Defintions that related in some way. This specification does not
```
Comment (deissnerk) [inline discovery/spec.md:70]: Definitions

File: discovery/spec.md [hunk 2]
```diff
@@ -114,6 +90,7 @@ defined:
 - Description: A unique identifier for the resource.
```
Comment (deissnerk) [inline discovery/spec.md:90]: Unique within what scope?

File: discovery/spec.md (hunk 1 above)
Comment (bigdavethewave) [reply discovery/spec.md:70]: LGTM

Comment (duglin) [reply discovery/spec.md:70]: fixed

File: discovery/spec.md (hunk 2 above)
Comment (duglin) [reply discovery/spec.md:90]: Is it ok that it says:
```
MUST be unique across all instances of the this resource type within the current Discovery Service.
```
down below?

Comment (sasha-tkachev) [reply discovery/spec.md:90]: Do we have a definition for the identity of the Discovery Service?

Comment (duglin) [reply discovery/spec.md:90]: @deissnerk or did you want me to repeat it in the Description line?

Comment (duglin) [reply discovery/spec.md:90]: @sasha-tkachev what do you mean by "definition for the identity" ? Below we say it must conform to RFC3986/3.3. Or are you asking for the semantic meaning behind the string? If so, I'm not sure we can get into that.

**Comment by duglin** [comment]
Approved on the 10/13 call - will submit a new PR if @deissnerk and/or @sasha-tkachev's comments need it.

**Pull request merged**